### ISSUE TITLE
Run many Playbooks from different Repositories

### ISSUE THREAD

**Issue by ITS1988** [ansible/awx#15281]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.
- [X] I am **NOT** reporting a (potential) security vulnerability. (These should be emailed to `security@ansible.com` instead.)

### Bug Summary

Hello everyone,

I have a question. We use Ansible AWX. We have an Ansible repository where we manage our playbooks that can be used by multiple projects and the project repositories where we keep the playbooks that contain project specific stuff.

We want to have this structure so that if something needs to be changed in the nodejs playbook, we don't have to make the change several times in several places, but only once in the playbook repository.

Is there any way other than using workflows that I can merge and run all playbooks that are needed for my project (e.g. Java17 from playbooks repo, Nodejs14 from playbooks repo and main playbook from project repo)?

Many thanks in advance!

### AWX version

17.0.1

### Select the relevant components

- [ ] UI
- [ ] UI (tech preview)
- [ ] API
- [ ] Docs
- [ ] Collection
- [ ] CLI
- [X] Other

### Installation method

docker development environment

### Modifications

no

### Ansible version

2.9.17

### Operating system

Rhel9

### Steps to reproduce

see summary

### Expected results

see summary

### Actual results

see summary

**Comment by djyasin** [comment]
Hello, this appears to be less of a bug report or feature request and more of a question. Could you please ask this on either the [Ansible AWX channel on Matrix](https://matrix.to/#/#awx:ansible.com) or the [Ansible Community Forum](https://forum.ansible.com/tag/awx)?

**Comment by djyasin** [comment]
Hello, we have not heard back from you in a while so we are going to go ahead and close this issue. Please feel free to open a new issue or reach out to us on the Forum should you need any additional assistance!
